README.md

```diff
@@ -272,7 +272,7 @@ List of all available rules. The rules ported from `golint` are left unchanged a
 | `function-result-limit` |  int | Specifies the maximum number of results a function can return    |    no    |  no   |
 | `imports-blacklist`   | []string | Disallows importing the specified packages                     |    no    |  no   |
 | `range-val-in-closure`|  n/a   | Warns if range value is used in a closure dispatched as goroutine|    no    |  no   |
-| `waitgroup-by-value  `|  n/a   | Warns on functions taking sync.WaitGroup as a by-value parameter |    no    |  no   |
+| `waitgroup-by-copy`   |  n/a   | Warns on functions taking sync.WaitGroup as a by-copy parameter  |    no    |  no   |
```

Comment:
Parameters are said to be passed by _value_ or by _reference_, therefore the documentation of the rule (`README.md`) **is correct** while the name of the rule is not OK (`rule/waitgroup-by-copy.go`)

I propose to set rule name to `waitgroup-by-value`.